LBHackney-IT/lbh-social-care-frontend
# added admin user

**What**
- renamed Admin group in Dev and added Admin group
- enabled feature flag only for dev user
- conditional render restricted visible only to Admin project
- extended user factories covering all the cases
- activated the new group in AWS

**How to test it**
- make sure that the "restricted" radio on create/edit user is only visible to Admin users
- make sure that the features behind feature flags (edit person, warning notes) are only visible to Dev users
